src/python/pants/bin/goal_runner.py

```diff
@@ -159,6 +157,15 @@ def _maybe_launch_pantsd(self, pantsd_launcher):
       with self._run_tracker.new_workunit(name='pantsd', labels=[WorkUnitLabel.SETUP]):
         pantsd_launcher.maybe_launch()
 
+  def _should_be_quiet(self, goals):
+    if self._explain:
+      return True
+
+    if self._global_options.get_rank('quiet') > RankedValue.HARDCODED:
```

Comment:
So this means that setting it in pants.ini will also force the behavior? (The PR description only mentions cmd line and env var).

Reply:
Right. PR title and description corrected.